<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newClient</class>
 <widget class="QDialog" name="newClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 255, 255);</string>
  </property>
  <widget class="QPlainTextEdit" name="FirstName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="plainText">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="LastName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="plainText">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="PhoneNumber">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="plainText">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Confirm">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 217, 217);</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
